### ISSUE TITLE
- UPDATE ControlNet to Work with Flux Checlpoints

### ISSUE THREAD

**Issue by danilomaiaweb** [lllyasviel/stable-diffusion-webui-forge#2491]
**Please UPDATE ControlNet to Work with Checkpoint Flux Dev or Schnell**
Some extremely amazing features do not work without the ControlNet
**inpaint, inpaint upload, Inpaint Anything**

SDForge is very amazing, but I can't use ControlNet together with Flux Checkpoints
Thank in Advance

**Comment by oumad** [comment]
Controlnets for flux aren't yet supported in forge, it's mentioned in the readme, but you can definitely still do inpaint in img2img tab, works fine for me using flux.

**Comment by danilomaiaweb** [comment]
> Controlnets para flux ainda não são suportados no forge, é mencionado no leia-me, mas você definitivamente ainda pode fazer inpaint na guia img2img, funciona bem para mim usando flux.

Inpaint actually works in the img2img tab, but without the use of ControlNet. If we use ControlNet, it presents errors. ControlNet allows us greater flexibility of resources and allows us to do incredible things, right?

We will wait for an update of ControlNet to work completely with Flux.

Thank for your reply my mate !

**Comment by oumad** [comment]
> Inpaint actually works in the img2img tab, but without the use of ControlNet. If we use ControlNet, it presents errors. ControlNet allows us greater flexibility of resources and allows us to do incredible things, right?

Yes Controlnets are definitely essential for more controllable/advanced workflows, but when it comes to inpainting at least, I was able to do decent inpaints with just the img2img tab in forge, you just need to adjust the denoise strength properly based on what you need to do.
The current ControlNets that exist for flux are based on flux dev which is not easily usable commercially, I would be more interested if we get something like this supported in forge instead : https://github.com/Yuanshi9815/OminiControl
